### PR TITLE
Use time.time() to make a timestamp, fixed #16

### LINKED ISSUE #16: Use time.time() to make a timestamp
The middleware adds a timestamp to the request which tells how long the request took. 
It's currently doing this by creating two `datetime.datetime.utcnow()` instances. 
It's much cheaper to use `time.time()` to make timestamps.

### PR DESCRIPTION
**Pull request by peterbe**
@mythmon @jezdez r?

### CODE REVIEW

File: src/dockerflow/django/middleware.py [hunk 1]
```diff
@@ -53,9 +53,11 @@ def _build_extra_meta(self, request):
                           request.user.pk or '')
         if hasattr(request, '_id'):
             out['rid'] = request._id
-        if hasattr(request, '_logging_start_dt'):
-            td = datetime.datetime.utcnow() - request._logging_start_dt
-            out['t'] = int(td.total_seconds() * 1000)  # in ms
+        if hasattr(request, '_logging_start_timestamp'):
+            # timestamp it took, in milliseconds
```
Comment (mythmon) [inline src/dockerflow/django/middleware.py]: I don't think "timestamp" is the right word to use here. Maybe "duration of request, in milliseconds"?

Comment (peterbe) [inline src/dockerflow/django/middleware.py]: oops

Comment (peterbe) [inline src/dockerflow/django/middleware.py]: fixed.

**Comment by mythmon** [comment]
LGTM, though I'll leave it to @jezdez to sign off on

**Comment by jezdez** [comment]
r+

**Pull request merged**